cdap-ui/app/features/hydrator/controllers/detail/top-panel-ctrl.js

```diff
@@ -34,9 +34,10 @@ angular.module(PKG.name + '.feature.hydrator')
       var lastRunDuration;
       var nextRunTime = DetailRunsStore.getNextRunTime();
       if (nextRunTime && nextRunTime.length) {
-        nextRunTime = nextRunTime[0].time? nextRunTime[0].time: null;
+        this.nextRunTime = nextRunTime[0].time? nextRunTime[0].time: null;
+      } else if (!nextRunTime || !nextRunTime.length) {
```

Comment:
nit - it seems like there's no point in putting the expression in this if condition.
It seems equivalent to just have this be an `else {`.